lib/bamboozled/api/base.rb

```diff
@@ -43,7 +43,10 @@ def request(method, path, options = {})
                 JSON.parse(response)
               end
             rescue
-              MultiXml.parse(response, symbolize_keys: true)
+              typecast = options.fetch(:typecast_values, true)
+              MultiXml.parse(response,
+                             symbolize_keys: true,
```

Comment:
Layout/AlignHash: Align the elements of a hash literal if they span more than one line.